integration-test-utils/ballerina/build.gradle

```
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+apply plugin: 'io.ballerina.plugin'
+
+buildscript {
+    repositories {
+//        mavenLocal()
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/plugin-gradle'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
+        }
+    }
+    dependencies {
+        classpath "io.ballerina:plugin-gradle:${project.ballerinaGradlePluginVersion}"
+    }
+}
+
+description = 'Ballerina - Observability Internal'
+
+def packageName = "testobserve"
+def packageOrg = "ballerina"
+
+def tomlVersion = stripBallerinaExtensionVersion("${project.version}")
+def ballerinaTomlFilePlaceHolder =
+        new File("${project.rootDir}/build-config/resources/integration-test-utils/Ballerina.toml")
+def ballerinaTomlFile = new File("$project.projectDir/Ballerina.toml")
+
+def stripBallerinaExtensionVersion(String extVersion) {
+    if (extVersion.matches(project.ext.timestampedVersionRegex)) {
+        def splitVersion = extVersion.split('-');
+        if (splitVersion.length > 3) {
+            def strippedValues = splitVersion[0..-4]
+            return strippedValues.join('-')
+        } else {
+            return extVersion
+        }
+    } else {
+        return extVersion.replace("${project.ext.snapshotVersion}", "")
+    }
+}
+
+
+ballerina {
+    packageOrganization = packageOrg
+    module = packageName
+    customTomlVersion ="0.0.0"
+    langVersion = ballerinaLangVersion
+}
+
+configurations {
+    externalJars
+}
+
+dependencies {
+    externalJars(group: 'io.netty', name: 'netty-codec-http', version: "${nettyCodecVersion}") {
+        transitive = false
+    }
+    externalJars(group: 'io.netty', name: 'netty-codec-http', version: "${gsonVersion}") {
```

Is this correct ? 🤔  "netty-codec-http" name and "gsonVersion" ? 🤔 
```suggestion
    externalJars(group: 'io.netty', name: 'netty-codec-http', version: "${gsonVersion}") {
```

---

Fixed with https://github.com/ballerina-platform/module-ballerinai-observe/pull/1/commits/1c9269cc5d6f67cda4563a526146f98e809fb35e